src/components/Accordion/PAccordionTrigger.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <AccordionHeader class="flex">
+    <AccordionTrigger
+      v-bind="delegatedProps"
+      class="p-accordion-trigger"
+      :class="props.class"
+    >
+      <slot />
+      <slot name="icon">
+        <p-icon
+          icon="ChevronDownIcon"
+          class="p-accordion-trigger-icon"
+        />
+      </slot>
+    </AccordionTrigger>
+  </AccordionHeader>
+</template>
+
+<script setup lang="ts">
+  import {
+    AccordionHeader,
+    AccordionTrigger,
+    type AccordionTriggerProps
+  } from 'radix-vue'
+  import { type HTMLAttributes, computed } from 'vue'
+
+  const props = defineProps<AccordionTriggerProps & { class?: HTMLAttributes['class'] }>()
+
+  const delegatedProps = computed(() => {
+    // eslint-disable-next-line id-length, no-unused-vars
+    const { class: _, ...delegated } = props
+
+    return delegated
+  })
+</script>
+
+<style scoped>
+.p-accordion-trigger {
+  @apply
+    cursor-pointer
+    flex
+    flex-1
+    items-center
+    justify-between
+    py-4
+    font-medium
+    transition-all
+    hover:underline
+    [&[data-state=open]>svg]:rotate-180
+}
+
+.p-accordion-trigger-icon {
+  @apply
+    h-4
+    w-4
+    shrink-0
+    transition-transform
+    duration-200
+}
+</style>
```

Comment:
```suggestion
<style scoped>
.p-accordion-trigger { @apply
    cursor-pointer
    flex
    flex-1
    items-center
    justify-between
    py-4
    font-medium
    transition-all
    hover:underline
    [&[data-state=open]>svg]:rotate-180
}

.p-accordion-trigger-icon { @apply
    h-4
    w-4
    shrink-0
    transition-transform
    duration-200
}
</style>
```

linting; also that data attribute svg selector is a little whack; can you convert that to a hopefully more readable class instead? something like:

```
.p-accordion-trigger[data-state=open] > svg { @apply
  rotate-180
}
```
i think that'll be a lot clearer to anyone reading this css

Reply:
+1 that feels like codegolf